Makefile

```diff
@@ -448,6 +448,11 @@ openstack: operator_namespace openstack_prep ## installs the operator, also runs
 	$(eval $(call vars,$@,openstack))
 	oc apply -f ${OPERATOR_DIR}
 
+.PHONY: openstack_wait
+openstack_wait: openstack ## waits openstack CSV to succeed.
+	$(eval $(call vars,$@,openstack))
+	timeout 300s bash -c 'until $$(oc get csv -l operators.coreos.com/openstack-operator.openstack-operators -n ${OPERATOR_NAMESPACE} | grep -q Succeeded); do sleep 1; done'
```

Comment:
Can we parameterized these timeout? Based on environment it might take more than 300 secs.